src/ScheduleRunCommand.js

```diff
@@ -0,0 +1,16 @@
+import './Command'
+
+export class ScheduleRunCommand extends Command {
+
+	name = 'schedule:run'
+	description = 'Starts the schedule daemon and executes commands as scheduled.'
+
+	run() {
+		this.info('Schedule daemon starting...')
+
+		this.cli.scheduler.start()
```

Comment:
Actually it looks like `Scheduler.start` will never resolve if there's no error, so we can just do:

```js
return this.cli.scheduler.start()
```